Description: Make update helper optional.
 According to TheAssassin the update helper is not yet ready in
 libappimage. Make it optional for now.
Author: TheAssassin
Origin: upstream
Last-Update: 2019-02-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: AppImageLauncher-1.0.0-beta-3/src/shared/CMakeLists.txt
===================================================================
--- AppImageLauncher-1.0.0-beta-3.orig/src/shared/CMakeLists.txt	2019-01-06 00:13:23.000000000 +0000
+++ AppImageLauncher-1.0.0-beta-3/src/shared/CMakeLists.txt	2019-01-06 13:47:28.403111240 +0000
@@ -1,5 +1,8 @@
 add_library(shared STATIC shared.h shared.cpp)
-target_link_libraries(shared PUBLIC PkgConfig::glib Qt5::Core Qt5::Widgets Qt5::DBus libappimage translationmanager trashbin libappimageupdate)
+target_link_libraries(shared PUBLIC PkgConfig::glib Qt5::Core Qt5::Widgets Qt5::DBus libappimage translationmanager trashbin)
+if(ENABLE_UPDATE_HELPER)
+  target_link_libraries(shared PUBLIC libappimageupdate)    
+endif()
 target_compile_definitions(shared
     PRIVATE -DCMAKE_INSTALL_PREFIX="${CMAKE_INSTALL_PREFIX}"
     PRIVATE -DCMAKE_INSTALL_LIBDIR="${CMAKE_INSTALL_LIBDIR}"
Index: AppImageLauncher-1.0.0-beta-3/src/shared/shared.cpp
===================================================================
--- AppImageLauncher-1.0.0-beta-3.orig/src/shared/shared.cpp	2019-01-06 00:13:23.000000000 +0000
+++ AppImageLauncher-1.0.0-beta-3/src/shared/shared.cpp	2019-01-06 13:44:29.090031065 +0000
@@ -26,8 +26,9 @@
 #include <QRegularExpression>
 #include <QSettings>
 #include <QStandardPaths>
+#ifdef ENABLE_UPDATE_HELPER
 #include <appimage/update.h>
-
+#endif
 // local headers
 #include "shared.h"
 #include "translationmanager.h"
